TestingApp/ViewController/ViewController.swift

```diff
@@ -333,7 +333,7 @@ extension ViewController {
             ) { result in
                 switch result {
                 case .success:
-                    self.entryWidget = Glia.sharedInstance.getEntryWidget(queueIds: [""])
+                    self.entryWidget = try? Glia.sharedInstance.getEntryWidget(queueIds: [""])
```

Comment:
Maybe error should not be ignored here? We can at least show some feedback regarding it.
```suggestion
               catchingError { [weak self] in
                    self?.entryWidget = try Glia.sharedInstance.getEntryWidget(queueIds: [""])
               }
```

Reply:
Agree. https://github.com/salemove/ios-sdk-widgets/pull/1066/commits/7efd8ef93c02f5b31d0bea788c38b7453cc83ed2